src/main/java/frc/robot/subsystems/Arm/ArmSubsystem.java

```diff
@@ -55,16 +55,26 @@ public void periodic() {
 
     pidOutput = MathUtil.clamp(pidOutput, -1, 1);
     armIO.setArmSpeed(pidOutput);
```

Comment:
Shouldn't this be eliminated? Should spin arm in the if statement that checked if the encoder is connected